stylesheets/all.scss

```diff
@@ -62,3 +62,16 @@ $screen-sm-min: 576px;
   justify-content: space-between;
   align-items: flex-end;
 }
+
+.cancelIcon {
```

Comment:
Can we avoid using global style for component-specific styling?

Css modules or css-in-js are better.